test/e2e/plugin_sync/plugin_sync_lifecycle_suite_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// pluginsynce2e provides plugin sync command specific E2E test cases
+package pluginsynce2e
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/vmware-tanzu/tanzu-cli/test/e2e/framework"
+	helper "github.com/vmware-tanzu/tanzu-cli/test/e2e/plugin_lifecycle"
+)
+
+func TestPluginSyncLifecycle(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Plugin-Sync-Lifecycle E2E Test Suite")
+}
+
+var (
+	tf                         *framework.Framework
+	e2eTestLocalCentralRepoURL string
+	pluginsSearchList          []*framework.PluginInfo
+	pluginGroups               []*framework.PluginGroup
+	pluginGroupToPluginListMap map[string][]*framework.PluginInfo
+)
+
+const CRDFilePath = "../framework/config/cli.tanzu.vmware.com_cliplugins.yaml"
```

Comment:
update the path to use `<repo-root>/apis/config/crd/bases/cli.tanzu.vmware.com_cliplugins.yaml`